docs/plugins/OxHugoFlavoredMarkdown.md

```diff
@@ -0,0 +1,29 @@
+---
+title: OxHugoFlavoredMarkdown
+tags:
+  - plugin/transformer
+---
+
+This plugin modifies Markdown content generated by [ox-hugo](https://github.com/kaushalmodi/ox-hugo) to make it compatible with Quartz, addressing specific formatting and syntax issues. See [[OxHugo compatibility]] for more information.
```

Comment:
```suggestion
This plugin provides support for [ox-hugo](https://github.com/kaushalmodi/ox-hugo) compatibility. See [[OxHugo compatibility]] for more information.

> [!important]
> This is mutually exclusive with [[Obsidian compatibility]].
```

Reply:
This warning duplicates the later warning:

> [!warning]
> While you can use this together with [[ObsidianFlavoredMarkdown]], it's not recommended because it might mutate the file in unexpected ways. Use with caution.
>
> If you use `toml` frontmatter, make sure to configure the [[Frontmatter]] plugin accordingly. See [[OxHugo compatibility]] for an example.

Which do you want to keep?

Reply:
oh I didn't realize you ahve this warning later. 

I think we should recommend users to use either or. Though they can be used together, let's not recommend it.

So I think it is fine to just say that these two are mutually exclusive?